charts/elasticsearch/templates/es-ingress.yaml

```
@@ -0,0 +1,46 @@
+########################################
+## Elasticsearch Logging Ingress ##
+########################################
+{{- if eq .Values.global.plane.mode "data" }}
+kind: Ingress
+apiVersion: {{ template "apiVersion.Ingress" . }}
+metadata:
+  name: {{ template "elasticsearch.fullname" . }}-ingress
+  labels:
+    tier: elasticsearch-networking
+    component: elasticsearch-logging-ingress
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    plane: {{ .Values.global.plane.mode }}
+  annotations:
+    kubernetes.io/ingress.class: "{{ .Release.Name }}-nginx"
+
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTP"
+    nginx.ingress.kubernetes.io/proxy-body-size: "100m"
+    nginx.ingress.kubernetes.io/proxy-read-timeout: "300"
+    nginx.ingress.kubernetes.io/proxy-send-timeout: "300"
+    nginx.ingress.kubernetes.io/proxy-buffer-size: "16k"
+    nginx.ingress.kubernetes.io/use-regex: "true"
+
+spec:
+  {{- if or .Values.global.tlsSecret .Values.global.acme }}
+  tls:
+  {{- if .Values.global.tlsSecret }}
+    - secretName: {{ .Values.global.tlsSecret }}
+  {{- end }}
```

global.acme is being removed. We don't need this condition.
```suggestion
  {{- if .Values.global.tlsSecret }}
  tls:
    - secretName: {{ .Values.global.tlsSecret }}
  {{- end }}
```